web/src/routes/backOffice/employees/components/EmployeeForm.svelte

```diff
@@ -0,0 +1,491 @@
+			"phoneNumber",
+		) as HTMLInputElement;
+		const locationInput = form.elements.namedItem(
+			"location",
+		) as HTMLInputElement;
+		const scheduleStartInput = form.elements.namedItem(
+			"scheduleEnd",
+		) as HTMLInputElement;
+		const scheduleEndInput = form.elements.namedItem(
+			"scheduleStart",
+		) as HTMLInputElement;
+
+		// Check if form is valid to prevent making a server request.
+		if (
+			!validateForm(
+				usernameInput.validity,
+				firstNameInput.validity,
+				lastNameInput.validity,
+				dateOfBirthInput.validity,
+				phoneNumberInput.validity,
+				locationInput.validity,
+				scheduleStartInput.validity,
+				scheduleEndInput.validity,
+				selectedCoordinate,
+			)
+		) {
+			return;
+		}
+
+		onSave(
+			username,
+			firstName,
+			lastName,
+			dateOfBirth,
+			phoneNumber,
+			{
+				type: "Feature",
+				geometry: {
+					type: "Point",
+					coordinates: selectedCoordinate,
+				},
+				properties: {},
+			},
+			scheduleStart,
+			scheduleEnd,
+		);
+	}
+</script>
+
+<form novalidate on:submit|preventDefault={handleSubmit}>
+	<DetailsHeader to={back} {title}>
+		<Link to={back} style="display:contents">
+			<Button variant="tertiary">{$t("cancel")}</Button>
+		</Link>
+		<Button type="submit" startIcon="check">{$t("save")}</Button>
+	</DetailsHeader>
+	<DetailsContent>
+		<DetailsSection label={$t("personalInfo")}>
+			<DetailsFields>
+				<!-- FirstName -->
+				<FormControl
+					label={$t("employees.firstName")}
+					error={!!formErrorMessages.firstName}
+					helperText={formErrorMessages.firstName}
+				>
+					<Input
+						required
+						name="firstName"
+						value={employee?.firstName}
+						error={!!formErrorMessages.firstName}
+						placeholder={$t("employees.firstName.placeholder")}
+						minLength={formFieldsLengths.firstName.min}
+						maxLength={formFieldsLengths.firstName.max}
+					/>
+				</FormControl>
+
+				<!-- LastName -->
+				<FormControl
+					label={$t("employees.lastName")}
+					error={!!formErrorMessages.lastName}
+					helperText={formErrorMessages.lastName}
+				>
+					<Input
+						required
+						name="lastName"
+						value={employee?.lastName}
+						error={!!formErrorMessages.lastName}
+						placeholder={$t("employees.firstName.placeholder")}
+						minLength={formFieldsLengths.firstName.min}
+						maxLength={formFieldsLengths.firstName.max}
+					/>
+				</FormControl>
+
+				<!-- Username -->
+				<FormControl
+					label={$t("employees.username")}
+					error={!!formErrorMessages.username}
+					helperText={formErrorMessages.username}
+				>
+					<Input
+						required
+						name="username"
+						value={employee?.username}
+						error={!!formErrorMessages.username}
+						placeholder={$t("employees.username.placeholder")}
+						minLength={formFieldsLengths.username.min}
+						maxLength={formFieldsLengths.username.max}
+					/>
+				</FormControl>
+
+				<!-- Birthdate -->
+				<FormControl
+					label={$t("employees.birthdate")}
```

Comment:
Just for consistency purposes, change every place that contains `birthdate` to `dateOfBirth`